addons/project/static/tests/project_tests.js

```diff
@@ -0,0 +1,130 @@
+odoo.define('project.project_kanban_tests', function (require) {
+"use strict";
+
+var KanbanView = require('web.KanbanView');
+var testUtils = require('web.test_utils');
+
+var createView = testUtils.createView;
+
+QUnit.module('project', {
+    beforeEach: function () {
+        this.data = {
+            'ir.attachment': {
+                fields: {
+                    name: {
+                        string: "Name",
+                        type: "char"
+                    },
+                },
+                records: [{
+                        id: 1,
+                        name: "1.png"
+                    },
+                    {
+                        id: 2,
+                        name: "2.png"
+                    },
+                ]
+            },
+            'project.task': {
+                fields: {
+                    name: {
+                        string: "Task Title",
+                        type: "char"
+                    },
+                    sequence: {
+                        string: "sequence",
+                        type: "integer"
+                    },
+                    displayed_image_id: {
+                        string: "cover",
+                        type: "many2one",
+                        relation: "ir.attachment"
+                    },
+                    kanban_state: {
+                        string: "State",
+                        type: "selection",
+                        selection: [
+                            ["abc", "ABC"],
+                            ["def", "DEF"],
+                            ["ghi", "GHI"]
+                        ]
+                    },
+                },
+                records: [{
+                        id: 1,
+                        name: "task1",
+                        sequence: 1,
+                        kanban_state: "abc"
+                    },
+                    {
+                        id: 2,
+                        name: "task2",
+                        sequence: 2,
+                        kanban_state: "abc"
+                    },
+                ]
+            },
+        };
+    }
+}, function () {
+    QUnit.module('image test');
+
+    QUnit.test('cover_image_test', function (assert) {
+        assert.expect(3);
+        var self=this;
```

Comment:
missing spaces: var self = this; instead of var self=this;

also, you should not need a reference to this.  Please remove this line